build-mbl/poky-test/poky-bitbake-wrapper.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright (c) 2019, Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+set -e
+set -u
+
+WORKAREA="$1"; shift;
+cd "${WORKAREA}/layers/poky"
+
+COMMAND="$1"; shift;
+
+set +u
+set +e
+# shellcheck disable=SC1091
+TEMPLATECONF=meta-poky/conf source oe-init-build-env
+
+set -e
+set -u
+
+$COMMAND
```

Comment:
I'd use "$@" here (and remove `COMMAND="$1"; shift`) so that words can be split properly.

Reply:
Changed (although still needed the shift to prevent the command being passed through the "source" chain)

Reply:
Oh, right yeah... This still doesn't fix the word splitting though. I suppose you'd need something like
```Shell
# Save command in array
COMMAND=("$@")
# Clear $@
set --

# Do stuff

# Run the command
"${COMMAND[@]}"
```

Then in the Python code that calls the script you'd pass each argument of the command as a separate element in the array passed to `subprocess.run`.